src/main/java/io/nats/client/impl/OrderedPullMessageManager.java

```diff
@@ -67,35 +72,4 @@ protected ManageResult manage(Message msg) {
 
         return manageStatus(msg);
     }
-
-    private void handleErrorCondition() {
-        try {
-            targetSid.set(null);
-            expectedExternalConsumerSeq = 1; // consumer always starts with consumer sequence 1
-
-            // 1. shutdown the manager, for instance stops heartbeat timers
-            shutdown();
-
-            // 2. re-subscribe. This means kill the sub then make a new one
-            //    New sub needs a new deliverSubject
-            String newDeliverSubject = sub.connection.createInbox();
-            sub.reSubscribe(newDeliverSubject);
-            targetSid.set(sub.getSID());
-
-            // 3. make a new consumer using the same deliver subject but
-            //    with a new starting point
-            ConsumerConfiguration userCC = js.nextOrderedConsumerConfiguration(originalCc, lastStreamSeq, newDeliverSubject);
-            js._createConsumerUnsubscribeOnException(stream, userCC, sub);
-
-            // 4. restart the manager.
-            startup(sub);
-        }
-        catch (Exception e) {
-            IllegalStateException ise = new IllegalStateException("Ordered subscription fatal error.", e);
-            js.conn.processException(ise);
-            if (syncMode) {
-                throw ise;
-            }
-        }
-    }
```

Comment:
This was based on the Push version where I had to maintain the same subscription handle that was given to the user, but it's done differently here and this is just more complicated since the subscription handle is kept from the user because they are using the simplification handle.